Sources/AblyChat/RoomLifecycleManager.swift

```diff
@@ -935,4 +982,94 @@ internal actor DefaultRoomLifecycleManager<Contributor: RoomLifecycleContributor
         // CHA-RL3g
         changeStatus(to: .released)
     }
+
+    // MARK: - RETRY operation
+
+    /// Implements CHA-RL5’s RETRY operation.
+    ///
+    /// - Parameters:
+    ///   - forcedOperationID: Allows tests to force the operation to have a given ID. In combination with the ``testsOnly_subscribeToOperationWaitEvents`` API, this allows tests to verify that one test-initiated operation is waiting for another test-initiated operation.
+    ///   - triggeringContributor: This is, in the language of CHA-RL5a, “the channel that became SUSPENDED”.
+    internal func performRetryOperation(testsOnly_forcingOperationID forcedOperationID: UUID? = nil, triggeredByContributor triggeringContributor: Contributor) async {
```

Comment:
Seems like it's not being called from anywhere apart from tests.

Reply:
Correct. See message of f08eabc9b76f5a81b92cdf4c6ef1c59789818005.

Reply:
Oops, sorry 🤦‍♂️